src/MergeContext.js

```diff
@@ -1652,7 +1657,8 @@ class PullRequest {
 
         await this._loadStaged();
         await this._loadRawPr(); // requires this._loadStaged()
-        await this._loadRequiredContexts();
+        this._contextsRequiredByGitHubConfigBase = await this._loadRequiredContexts(this._prBaseBranch());
+        this._contextsRequiredByGitHubConfigStaging = await this._loadRequiredContexts(Config.stagingBranch());
```

Comment:
Please rename PR's _loadRequiredContexts() to _loadRequiredContextsFor() and then move these two assignments into  _loadRequiredContexts() function (that will do nothing else but set these two fields). Doing so will effectively undo PR change this comment is attached to, restoring this code original "style", and restoring synchronization with the "requires..." comment below.

```suggestion
        await this._loadRequiredContexts();
```

Reply:
> Please rename PR's _loadRequiredContexts()

Done.